python/cuml/test/test_pickle.py

```diff
@@ -65,9 +68,22 @@
     cuml.neighbors.NearestNeighbors])
 k_neighbors_models = k_neighbors_config.get_models()
 
-unfit_pickle_xfail = ['ARIMA', 'KalmanFilter', 'ForestInference']
-unfit_clone_xfail = ['ARIMA', 'ExponentialSmoothing', 'KalmanFilter',
-                     'MBSGDClassifier', 'MBSGDRegressor']
+unfit_pickle_xfail = [
+    'ARIMA',
+    'AutoARIMA',
+    'KalmanFilter',
+    'BaseRandomForestModel',
+    'ForestInference'
+]
+unfit_clone_xfail = [
+    'ARIMA',
+    'AutoARIMA',
+    "BaseRandomForestModel",
+    'ExponentialSmoothing',
+    'KalmanFilter',
+    'MBSGDClassifier',
```

Comment:
why do these fail? Because the nested solver is not cloned properly?

Reply:
Most of these fail because because of 2 reasons: 1) They are a base class and not meant to be instantiated directly (and really should be abstract) or 2) They have constructors that require some non-default arguments which doesnt work in this test. For example, ARIMA needs an array for the parameter `endog`.

Its easier to understand with code. The clone test is essentially doing this:
```python
# Create model with no arguments
old_model = all_models[model_name]()

# Create a new model using old_model's params
new_model = all_models[model_name](**old_model.get_params())

# Check that the params are the same
assert new_model.get_params() == old_model.get_params()
```

So models that require an argument to be instantiated would fail here: `old_model = all_models[model_name]()`